tinyms/data/utils.py

```diff
@@ -18,8 +18,33 @@
 import gzip
 import tarfile
 import requests
+from PIL import Image
+import numpy as np
+from mindspore import Tensor
 
-__all__ = ['download_dataset']
+__all__ = ['download_dataset', 'generate_image_list', 'load_resized_img', 'load_img']
```

Comment:
The functions is not suitable to be defined in `data` module, you can place them at `tinyms/vision/utils.py` file.